.github/dependabot.yml

```diff
@@ -7,3 +7,10 @@ updates:
     labels:
     - "skip-release-notes"
     - "dependencies"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule: 
+      interval: weekly
+    labels: 
+     - "skip-release-notes"
+     - "dependencies"
```

Comment:
Nit: can we add a trailing newline here?

Reply:
+1

Reply:
done 👍🏻  🎉